workspace.json

```diff
@@ -2850,6 +2850,17 @@
             "jestConfig": "libs/service-portal/documents/jest.config.js",
             "passWithNoTests": true
           }
+        },
+        "extract-strings": {
```

Comment:
These targets are to be invoked manually, right?

Reply:
Yes :)